tests/main/interfaces-system-observe/task.yaml

```diff
@@ -42,26 +42,29 @@ execute: |
         su -l -c "test-snapd-system-observe-consumer.dbus-introspect" test | MATCH "$expected"
     fi
 
-    if [ "$(snap debug confinement)" = strict ] ; then
-        echo "And the policy has the ptrace suppression rule"
-        MATCH '^deny ptrace \(trace\),' < /var/lib/snapd/apparmor/profiles/snap.test-snapd-system-observe-consumer.consumer
+    if [ "$(snap debug confinement)" = partial ] ; then
```

Comment:
Mentioned this in another review, but we'll probably need to start looking at `snap debug sandbox-features` and making sure that apparmor profile is not downgraded. This way we can check the confinement on systems where AppArmor is supported by snapd.